docs/application-connector/docs/040-cr-remote-evironment.md

```diff
@@ -55,17 +51,18 @@ This table lists all the possible parameters of a given resource together with t
 |:----------:|:-------------:|:------|
 | **metadata.name** |    **YES**   | Specifies the name of the CR. |
 | **spec.source** |    **NO**   | Identifies the Remote Environment in the cluster. |
-| **spec.source.environment** |    **YES**   | Specifies the environment of the connected Remote Environment. |
-| **spec.source.type** |    **YES**   | Specifies the type of the connected Remote Environment. |
-| **spec.source.namespace** |    **YES**   | Specifies the namespace of the connected Remote Environment. |
 | **spec.description** |    **NO**   | Describes the connected Remote Environment.  |
 | **spec.accessLabel** |    **NO**   | Labels the environment when the [EnvironmentMapping](041-cr-environment-mapping.md) is created. |
+| **spec.labels** |    **NO**   | Labels indentified Remote Environment taxonomy.  |
 | **spec.services** |    **NO**   | Contains all services that the Remote Environment provides. |
 | **spec.services.id** |    **YES**   | Identifies the service that the Remote Environment provides. |
+| **spec.services.identifier** |    **NO**   | Additional identifier of the service class. |
```

Comment:
Regarding structure: services also have `name` field (used by service-catalog I think), it is filled by metadata service during registration.